javascript/apis/third-party-apis/nytimes/index.html

```diff
@@ -44,137 +44,40 @@ <h1>NY Times article search</h1>
-
-        if(startDate.value !== '') {
-          url += '&begin_date=' + startDate.value;
-        };
-
-        if(endDate.value !== '') {
-          url += '&end_date=' + endDate.value;
-        };
-
-        // Use fetch() to make the request to the API
-        fetch(url).then(function(result) {
-          return result.json();
-        }).then(function(json) {
-          displayResults(json);
-        });
-      }
-
-      function displayResults(json) {
-        while (section.firstChild) {
-            section.removeChild(section.firstChild);
-        }
-
-        var articles = json.response.docs;
-
-        if(articles.length === 10) {
-          nav.style.display = 'block';
-        } else {
-          nav.style.display = 'none';
-        }
-
-        if(articles.length === 0) {
-          var para = document.createElement('p');
-          para.textContent = 'No results returned.'
-          section.appendChild(para);
-        } else {
-          for(var i = 0; i < articles.length; i++) {
-            var article = document.createElement('article');
-            var heading = document.createElement('h2');
-            var link = document.createElement('a');
-            var img = document.createElement('img');
-            var para1 = document.createElement('p');
-            var para2 = document.createElement('p');
-            var clearfix = document.createElement('div');
-
-            var current = articles[i];
-            console.log(current);
-
-            link.href = current.web_url;
-            link.textContent = current.headline.main;
-            para1.textContent = current.snippet;
-            para2.textContent = 'Keywords: ';
-            for(var j = 0; j < current.keywords.length; j++) {
-              var span = document.createElement('span');
-              span.textContent = current.keywords[j].value + ' ';
-              para2.appendChild(span);
-            }
-
-            if(current.multimedia.length > 0) {
-              img.src = 'http://www.nytimes.com/' + current.multimedia[0].url;
-              img.alt = current.headline.main;
-            }
-
-            clearfix.setAttribute('class','clearfix');
-
-            article.appendChild(heading);
-            heading.appendChild(link);
-            article.appendChild(img);
-            article.appendChild(para1);
-            article.appendChild(para2);
-            article.appendChild(clearfix);
-            section.appendChild(article);
-          }
-        }
-      };
-
-      function nextPage(e) {
-        pageNumber++;
-        fetchResults(e);
-      };
-
-      function previousPage(e) {
-        if(pageNumber > 0) {
-          pageNumber--;
-        } else {
-          return;
-        }
-        fetchResults(e);
-      };
+    // Defining a baseURL and key to as part of the request URL
+
+const baseURL = 'https://api.nytimes.com/svc/search/v2/articlesearch.json';
+const key = 'INSERT-YOUR-API-KEY-HERE';
```

Comment:
Why not indented

Reply:
Sorry copied from another local virtual host where I was testing. Fixed and did another PR

Reply:
The new pull request is named nytimes-var does it matter it is a different name?

Reply:
I fixed the indenting, hopefully I haven't messed up again